src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/MessageProcessorFunctionalTest.java

```diff
@@ -60,8 +73,8 @@ public void should_process_message_with_the_lowest_event_timestamp_for_that_case
         });
 
         await().ignoreException(AssertionError.class)
-            .pollInterval(3, SECONDS)
-            .atMost(120, SECONDS)
+            .pollInterval(2, SECONDS)
```

Comment:
Again, make these values constants - probably somewhere common outside of the test classes themselves, to make fine tuning the values a simpler change